README.md

```diff
@@ -1,5 +1,7 @@
-# Crossterm | cross-platform terminal manipulating library.
- [![Donate](https://img.shields.io/badge/Donate-PayPal-green.svg)](https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=Z8QK6XU749JB2) ![Travis][s7] ![Workflow][s8] [![Latest Version][s1]][l1] [![MIT][s2]][l2] [![docs][s3]][l3] ![Lines of Code][s6] [![Join us on Discord][s5]][l5]
+<img align="center" style="margin-bottom=10px; height=255px" src="docs/crossterm_full.png" />
```

Comment:
Use the full URL here. Two phases:

* use the `0.11` branch to test it, how it looks like, etc.
* then, just before merging, replace it with the `master` branch

Reason? Same README.md is used on [crates.io](https://crates.io/crates/crossterm) and there's nothing like `docs/crossterm_full.png`.